code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -209,6 +209,24 @@
 		/obj/item/clothing/head/collectable/slime,  // 0TC priceless
 		/obj/item/encryptionkey/syndicate) // 10TC
 
+/// 150 TC
+
+/obj/item/storage/box/syndie_kit/bundle/tf13_spy
+	name = "TF13 Spy"
+	desc = "Brush up on your French and hone your backstabbing skills like a true, suave Gentlemann."
+	items = list(
+		/obj/item/gun/projectile/revolver, // 65TC
+		/obj/item/ammo_box/a357, // 15TC
+		/obj/item/ammo_box/a357, // 15TC
+		/obj/item/pen/edagger, // 10TC
+		/obj/item/chameleon, // 25TC
```

Comment:
with the suggestion of the cham kit, i'd also think the stealth box would make more sense

Reply:
Bugger me, the box was what I had in mind when brainstorming last night and I forgot about it